src/js/components/ServiceDetail.js

```diff
@@ -8,11 +9,17 @@ import ServiceDetailConfigurationTab from './ServiceDetailConfigurationTab';
 import ServiceDetailTaskTab from './ServiceDetailTaskTab';
 import ServiceFormModal from './modals/ServiceFormModal';
 import ServiceInfo from './ServiceInfo';
+import ServiceScaleFormModal from './modals/ServiceScaleFormModal';
 import ServicesBreadcrumb from './ServicesBreadcrumb';
 import TabsMixin from '../mixins/TabsMixin';
 
 const METHODS_TO_BIND = [
   'onActionsItemSelection',
+  'onAcceptDestroyConfirmDialog',
+  'onAcceptSuspendConfirmDialog',
+  'onCancelDestroyConfirmDialog',
+  'onCancelSuspendConfirmDialog',
+  'onCloseScaleFormModal',
```

Comment:
I think the wording for closing a _Modal_ is `close`, we use it like that also at other places.
On the other hand the wording in the _Confirm_ is `cancel`.
So if I change this, the things will be mixed up at another place.
I would like to stick with this.